frontend/public/components/monitoring/alert-manager-config.tsx

```diff
@@ -97,7 +270,25 @@ type AlertManagerConfigurationProps = {
   onCancel?: () => void;
 };
 
-type AlertRoutingProps = {
+type AlertManagerProps = {
```

Comment:
Maybe better to split into `AlertRoutingProps` and `ReceiversProps`?

Reply:
yes, did that